modules/rhel-compute-updating.adoc

```diff
@@ -53,9 +51,9 @@ By default, the base OS RHEL with "Minimal" installation option enables firewall
 +
 [source,terminal]
 ----
-# subscription-manager repos --disable=rhel-7-server-ose-4.8-rpms \
-                             --enable=rhel-7-server-ansible-2.9-rpms \
-                             --enable=rhel-7-server-ose-4.9-rpms
+# subscription-manager repos --disable=rhel-8-server-ose-4.9-rpms \
```

Comment:
As the above comment, looks like we still need RHEL-7 machine to run the Ansible playbook, so we may need to keep this as
```
# subscription-manager repos \
    --enable="rhel-7-server-rpms" \
    --enable="rhel-7-server-extras-rpms" \
    --enable="rhel-7-server-ansible-2.9-rpms" \
    --enable="rhel-7-server-ose-4.10-rpms"
```